### PR TITLE
Added the api endpoints

### PR DESCRIPTION
**Pull request by munezerobagira**

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const Post = require("./models/Post");
+const router = express.Router();
+router.get("/posts", async (req, res) => {
+  const posts = await Post.find();
```
Comment (habaumugisha1) [inline routes.js:5]: separate routers and app logic

**Review by habaumugisha1 — COMMENTED** [review]
Please use ES6 instead of ES5

File: src/routes.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import express from "express";
+import Post from "./models/Post";
+const router = express.Router();
+router.get("/posts", async (req, res) => {
+  const posts = await Post.find();
```
Comment (habaumugisha1) [inline src/routes.js:5]: Don't use business login in the routes

**Comment by munezerobagira** [comment]
Working on it

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,2 @@
+MONGO_URL=mongodb://localhost:27017/acmedb
```
Comment (habaumugisha1) [inline .env.example:1]: don't add values here, it keeps only variable

File: src/services/PostService.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  //create a post
+
+  static async createPost({ title, content }) {
+    try {
+      const post = new Post({
+        title,
+        content,
+      });
+      await post.save();
+      return { post };
```
Comment (habaumugisha1) [inline src/services/PostService.js:68]: please return json object

**Pull request merged**